.github/workflows/release.yaml

```diff
@@ -0,0 +1,67 @@
+name: release
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
```

Comment:
with this pattern, we won't be able to create a release candidate like `v0.11.0-rc0`. I didn't find the way to have an additional pattern with the github action pattern syntax: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
Would be cool to find the way, so we don't have to think again about it